content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/150-introspection.mdx

```diff
@@ -497,24 +502,26 @@ The data model now looks similar to this (note that the fields on the models hav
 model Post {
   id        Int      @id @default(autoincrement())
   title     String   @db.VarChar(255)
-  createdAt DateTime @default(now()) @db.Timestamp(6)
-  content   String?
+  createdAt DateTime @default(now()) @db.Timestamp(0)
+  content   String?  @db.Text
   published Boolean  @default(false)
   authorId  Int
-  User      User     @relation(fields: [authorId], references: [id])
+  User      User     @relation(fields: [authorId], references: [id], onDelete: NoAction, onUpdate: NoAction, map: "Post_ibfk_1")
+
+  @@index([authorId], map: "authorId")
 }
 
 model Profile {
   id     Int     @id @default(autoincrement())
-  bio    String?
-  userId Int     @unique
-  User   User    @relation(fields: [userId], references: [id])
+  bio    String? @db.Text
+  userId Int     @unique(map: "userId")
+  User   User    @relation(fields: [userId], references: [id], onDelete: NoAction, onUpdate: NoAction, map: "Profile_ibfk_1")
 }
 
 model User {
   id      Int      @id @default(autoincrement())
   name    String?  @db.VarChar(255)
-  email   String   @unique @db.VarChar(255)
+  email   String   @unique(map: "email") @db.VarChar(255)
   Post    Post[]
```

Comment:
We could make this whole Introspction result "cleaner" by adapting the FK and unique index names in the underlying SQL to our defaults - then the `map` would not have to be rendered here.

(Same could even be true for some of the `@db.Text` that are added)

Reply:
Thanks for the suggestion. I'll do the same for the previous one with the `db.Timestamp(0)` to use `@db.Timestamp(6)` instead.

Reply:
> We could make this whole Introspction result "cleaner" by adapting the FK and unique index names in the underlying SQL to our defaults - then the map would not have to be rendered here.
> (Same could even be true for some of the @db.Text that are added)

I'm a bit torn about this. On the one hand, I am all for making things cleaner. On the other hand, this is supposed to show the scenario for an existing DB in which (I'd assume) it's unlikely that people have been using our own defaults. In that sense, it might be more instructive if the introspection result actually contains some of these extra annotations.

Reply:
For now, I'll leave the introspected schema as is and add a note about the attributes & attribute functions added to the schema.